src/main/java/Event/Task.java

```diff
@@ -0,0 +1,45 @@
+package Event;
+
+public class Task {
+    static String BREAK_LINE = "____________________________________________________________";
+    protected String description; //Holds the textual description of the task.
```

Comment:
You could reconsider the placement of comments to the line before the commented code to adhere to the java coding [standards](https://se-education.org/guides/conventions/java/basic.html)